packages/dds/tree/src/feature-libraries/chunked-forest/basicChunk.ts

```diff
@@ -53,6 +53,16 @@ export class BasicChunk extends ReferenceCountedBase implements TreeChunk {
 		super();
 	}
 
+	private _summaryRefId: string | undefined;
```

Comment:
Given TreeChunk is intended to by copy on write, and not tied to a specific SharedTree instance, this pattern seems concerning.

If a given TreeChunk is un use in two distinct shared trees, I suspect this wont work.

Also if kept, this should be documented: all properties, especially mutable ones, should get documented.

Reply:
To avoid possible collision issues while allowing using very short keys when actually blobs in summaries, and to solve the copy on write issues with mutation, I think it might make more sense to instead store the actual `FieldUnchanged` here instead.

That would work well with the proposed tryReuseBlob API I suggested in https://github.com/microsoft/FluidFramework/pull/24949/files#r2178536725